tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -595,14 +595,15 @@ public void deleteEmptyDirectoryTest() throws Exception {
   public void deleteFileTest() throws Exception {
     long fileId = mFsMaster.createFile(new AlluxioURI("/testFile"), CreateFileOptions.defaults());
     Assert.assertEquals(fileId, mFsMaster.getFileId(new AlluxioURI("/testFile")));
-    Assert.assertTrue(mFsMaster.delete(new AlluxioURI("/testFile"), true));
+    mFsMaster.delete(new AlluxioURI("/testFile"), true);
     Assert.assertEquals(IdUtils.INVALID_FILE_ID, mFsMaster.getFileId(new AlluxioURI("/testFile")));
   }
 
   @Test
   public void deleteRootTest() throws Exception {
-    Assert.assertFalse(mFsMaster.delete(new AlluxioURI("/"), true));
-    Assert.assertFalse(mFsMaster.delete(new AlluxioURI("/"), false));
+    mThrown.expect(InvalidPathException.class);
```

Comment:
aside: we should only use one idiom for catching exceptions ... and since the ExpectedException idiom can interfere with WhiteBox, we should use the explicit try-catch idiom consistently ... this is something for your "unit test story"